.env.ci

```diff
@@ -59,3 +58,4 @@ KRATOS_MASTER_USER_PASSWORD=${KRATOS_MASTER_USER_PASSWORD}
 KRATOS_PUBLIC_API=http://kratos:4433
 KRATOS_ADMIN_API=http://kratos:4434
 KRATOS_CALLBACK_API_KEY=${KRATOS_CALLBACK_API_KEY}
+MONGODB_VERSION=${MONGODB_VERSION}
```

Comment:
This should be hard coded or omitted if we can set a default directly in the docker-compose file.

Reply:
Need to check if the default works